k8s/elasticsearch.yaml

```diff
@@ -46,6 +91,9 @@ spec:
               mountPath: /usr/share/elasticsearch/data
             - name: es-snapshots
               mountPath: /mnt/snapshots
+            - name: es-config
+              mountPath: /usr/share/elasticsearch/config/elasticsearch.keystore
+              subPath: elasticsearch.keystore
```

Comment:
You define a "es-config" file config mount but you also defined a "es-config" volume in the volumeClaimTemplates?